docs/configure

```diff
@@ -492,7 +487,6 @@ dnslookup:
   domains = ! +local_domains
   transport = remote_smtp
   ignore_target_hosts = 0.0.0.0 : 127.0.0.0/8
-  headers_remove = X-Spam-Report:X-Spam-Score
```

Comment:
I found more about the headers (https://github.com/Exim/exim/wiki/AclSmtpData), so we could use:
````
condition       = ${if def:h_x-spam-flag: {yes}{no}}
````
this only checks if the header is present, then we would need a second condition:
````
${if bool{$h_x-spam-flag}}
````
perhaps it's just enough to use:
````
${if bool_lax{$h_x-spam-flag}}
````
http://www.exim.org/exim-html-current/doc/html/spec_html/ch-string_expansions.html